src/test/java/spec/yandexSpec/CreateAccount.java

```diff
@@ -18,46 +18,49 @@
 
 public class CreateAccount extends BaseSpec {
```

Comment:
Create Account? Don't see tests, which creates account

Reply:
renamed to YandexMailSpecs